lib/greet.js

```diff
@@ -0,0 +1,10 @@
+var myObj = {
+  greet: function greet(name) {
+    console.log('hello ' + name);
+    return ('hello ' + name);
+  },
+};
+
+myObj.greet(process.argv[2]); //gimme thems bonus points!
```

Comment:
This is almost perfect! 

This line though should be located in an `index.js` entry point file that's located in the root of the project dir (same level as package.json).